### PR TITLE
integration test: Json queries

### PR DESCRIPTION
**Pull request by cendhu**

### CODE REVIEW

File: test/queries/json_queries_test.go [hunk 1]
```diff
@@ -0,0 +1,453 @@
...
+	"time"
+
+	"github.com/hyperledger-labs/orion-server/pkg/constants"
+	"github.com/hyperledger-labs/orion-server/pkg/server/testutils"
+	"github.com/hyperledger-labs/orion-server/pkg/types"
+	"github.com/hyperledger-labs/orion-server/test/setup"
+	"github.com/stretchr/testify/require"
+)
+
+func TestJSONQueries(t *testing.T) {
```
Comment (tock-ibm) [inline test/queries/json_queries_test.go:18]: Test cases here look good. I am missing some negative/corner cases though.
- bad selector
- JSON query on a db w/o index
- an empty result set
- JSON query on db the user has no read permission
etc.

Comment (cendhu) [reply test/queries/json_queries_test.go:18]: Done

**Review by tock-ibm — APPROVED** [review]
